src/panel_state_manager.cpp

```diff
@@ -1096,6 +1098,19 @@ void PanelStateManager::setCEState()
     }
 }
 
+bool PanelStateManager::isFunctionSupported(const types::FunctionNumber funcNum)
+{
+    std::vector<types::FunctionNumber> supportedFuncs = {21, 22, 34, 65, 66,
+                                                         67, 68, 69, 70};
+
+    auto pos = find(supportedFuncs.begin(), supportedFuncs.end(), funcNum);
```

Comment:
Can get rid of an extra variable as knowing position in this case is not required for any processing further.
if(find(supportedFuncs.begin(), supportedFuncs.end(), funcNum) != supportedFuncs.end()) ?